functions.py

```diff
@@ -217,6 +215,11 @@ def mining_behaviour(tx1, ty1, tx2, ty2, mr_start, mr_end, ml_start, ml_end, rm_
         pyautogui.click(button='left')
         pyautogui.press('f2')
 
+        # move towards target 1
+        sleep_and_log(3)

```

Comment:
this causes the ship to move away from the target roads causing them to move down the list. Thus I need to find a way when mining script resets, to single click the roid before hitting f1/2 (activating laser). This will in worst case cause many targeted roids, however, the chance of using up all 5 lock targets is slim

Reply:
eve supports unlock all targets. But shortcut needs to be bound. So I suggest adding an option in the bot to configure in properties a key combo for deactivating all targets and that it will use this combo if configured, if not using old behaviour.

Reply:
I have bound it to Ctrl-alt-t on my setup, and will test later today

Reply:
rgr, tell me when its rdy to merge.

Reply:
going to test right after server reset today. Made a couple of changes that hopefully will make unlock all targets work. Also reverted a lot of the unrelated changes to remove "debugging noise". 

Btw use of this thing must be monitored, I keep a local chat tab open so I can see if people are trying to tag me as bot. A good counter measure is to answer requests to join fleet and politely decline. Or, stop the bot program and warp out after the current mining session.

Reply:
finally made the Auto Unlock of all targets work. Now it will "follow" the closest asteroid and never keep more than two locks